Tests/UITests/TestApp/OnboardingTestsView.swift

```diff
@@ -15,42 +15,41 @@ struct OnboardingTestsView: View {
     @Binding var onboardingFlowComplete: Bool
     @State var showConditionalView = false
 
-
     var body: some View {
-        OnboardingStack(onboardingFlowComplete: $onboardingFlowComplete) {
+        OnboardingStack(didComplete: $onboardingFlowComplete) {
             OnboardingStartTestView(
                 showConditionalView: $showConditionalView
             )
             OnboardingWelcomeTestView()
             OnboardingSequentialTestView()
-
+            
             OnboardingConsentTestView(
                 consentTitle: "First Consent",
                 consentText: "This is the first *markdown* **example**",
                 documentIdentifier: ConsentDocumentIdentifiers.first
             )
-
+            
             OnboardingConsentFinishedRenderedView(
                 consentTitle: "First Consent",
                 documentIdentifier: ConsentDocumentIdentifiers.first
             )
-
+            
             OnboardingConsentTestView(
                 consentTitle: "Second Consent",
                 consentText: "This is the second *markdown* **example**",
                 documentIdentifier: ConsentDocumentIdentifiers.second
             )
-                .onboardingIdentifier(ConsentDocumentIdentifiers.second)
+            .onboardingIdentifier(ConsentDocumentIdentifiers.second)
```

Comment:
We use a different indentation standard across Spezi, would be great to stay consistent.

Reply:
Xcode defaults these to the non-indented version (it used to indent in cases like this; i had assumed that's why different parts of the codebase use different indentation for view modifiers following multi-line function calls)

Reply:
Yep agreed, but my suggested indentation is the standard in Spezi, we should stick to it, even tho it's a bit painful sometimes with Xcode